datadog-checks-base/datadog_checks/utils/subprocess_output.py

```diff
@@ -0,0 +1,34 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import logging
+
+try:
+    from _util import get_subprocess_output as subprocess_output
```

Comment:
So `_util` is a module provided by the agent6 bindings, and we definitely want this... But I think I would do this maybe slightly differently to play better with agent5. We could maybe have (yet) another nested `try...except ImportError`  we try to import Agent5 first, then fallback to trying to load this off agent 6. It wouldn't be too pretty, but it would probably provide the backward compatibility we'd need.

```
try:
    try:
        # Agent5
        from utils.subprocess_output import get_subprocess_output as subprocess_output
    except ImportError:
        #Agent6
        from _util import get_subprocess_output as subprocess_output
except ImportError:
    from ..stubs._util import subprocess_output 
```
we could also use a flag to perhaps reflact if we actually loaded the agent5, agent6 or stub code if we need to do anything different in the local `get_subprocess_output`